test/integration/authorization-test.ts

```diff
@@ -27,6 +27,30 @@ describe('authorization headers', function() {
     });
   });
 
+  describe('when header is set in a custom generateAuthHeader', function() {
+    let originalHeaderFn = Person.generateAuthHeader;
+    beforeEach(function() {
+      ApplicationRecord.jwt = 'cu570m70k3n';
+      Person.generateAuthHeader = function(token) {
+        return `Bearer ${token}`;
+      };
+    });
+
+    afterEach(function() {
+      fetchMock.restore();
+      Person.generateAuthHeader = originalHeaderFn;
+    });
+
+    it('expects true to be true', function(done) {
```

Comment:
let's make this a meaningful test description please

Reply:
🤦‍♂️ @richmolj sorry about that. I was struggling with `fetchMock` and started by clearing any chance to have an error on the setup. Thank you. Sorry. And thank you again.